src/amo/utils/addonExtract.ts

```diff
@@ -23,16 +24,9 @@ export async function extractAddon(
     }
   }
 
-  const extract = require("extract-zip");
-  await extract(
-    compressedFilePath,
-    { dir: workspaceFolder + "/" + addonGUID + "/" + addonVersion },
-    function (err: any) {
-      if (err) {
-        console.log(err);
```

Comment:
you've lost some logging along the way?

Reply:
Eslint required the "require" statement to be an import. Upon that, it said that only 2 Params were required and gave an error for the function.